iiod/iiod.c

```diff
@@ -407,9 +407,24 @@ static int main_interactive(struct iio_context *ctx, bool verbose, bool use_aio)
 
 	if (!use_aio) {
 		flags = fcntl(STDIN_FILENO, F_GETFL);
-		fcntl(STDIN_FILENO, F_SETFL, flags | O_NONBLOCK);
+		if (flags > 0)
```

Comment:
is `flags >= 0`  also valid? i.e. `flags` being also zero;

Reply:
I'm not sure if that is practical for STDIN (it should be open as write), but in theory, yes - it's not an error, so I updated. Good catch. Thanks